### ISSUE TITLE
Refactor: conditional classnames

### ISSUE THREAD

**Issue by unsky559** [unsky559/mathBattle#54]
Refactor conditional classnames. Use [classnames](https://github.com/JedWatson/classnames)

```javascript
<div className={["avatar", props.big ? "big" : ""].join(" ")}> // TODO: refactor classNames
```

Appear at: 
- `src/components/avatar/avatar.tsx`
- `src/components/gameKey/gameKey.tsx`
- `src/components/logo/logo.tsx`

**Comment by unsky559** [comment]
Closed at #55

**Comment by unsky559** [comment]
Closed at #55
